Makefile

```diff
@@ -637,3 +637,9 @@ get-phony:
 get-silent:
 	echo ".SILENT:" > make.silent.targets
 	$(foreach target,$(MUFFLED_TARGETS),echo $(target) >> make.silent.targets;)
+
+extract_assets: $(SOTNASSETS)
+	cd tools/sotn-assets; $(GO) install
+	$(SOTNASSETS) extract config/assets.$(VERSION).yaml
+build_assets: $(SOTNASSETS)
+	$(SOTNASSETS) build config/assets.$(VERSION).yaml
```

Comment:
@ProjectOblivion can you please have a look and make sure these Makefile changes fit with your stuff? I moved this out of `Makefile.psx.mk` and just put it at the bottom of the file but I can move it elsewhere if you let me know where it should go.

Also see changes in the PSP Makefile to add assets steps

Reply:
Done 👍